lib/topic.js

```diff
@@ -141,7 +141,10 @@ TopicMQ.prototype.publish = function (topic, content, options) {
   options.persistent = options.persistent || false;
   options.timestamp = options.timestamp || Date.now();
 
+  const content_string = (typeof content === 'object') ?
+    JSON.stringify(content) : content; 
```

Comment:
`JSON.stringify` can throw an error, do you think we should catch them or leave it as it is?